### PR TITLE
Remove material-dashboard dependency #56

### PR DESCRIPTION
**Pull request by belkassaby**
*remove material-dashboard.scss
*Use Studyintake platform UI and scss

### CODE REVIEW

**Comment by belkassaby** [comment]
This PR is live at http://34.23.249.48/

**Comment by belkassaby** [comment]
I also added #91 to this PR as there were files that had too much code changes. live app can be seen here : [35.227.57.161](http://35.227.57.161:80 )

**Review by iimpulse — APPROVED** [review]

**Review by ielis — APPROVED** [review]
The app looks great, thanks a lot!

File: frontend/package.json [hunk 1]
```diff
@@ -32,16 +33,21 @@
     "@angular/router": "^14.2.0",
     "@fortawesome/fontawesome-free": "~5.6.3",
     "@types/chart.js": "^2.7.40",
+    "ProtVista": "^2.0.13",
```
[the remark this answers is not in the document]
Comment (belkassaby) [reply frontend/package.json]: no, it seems i added it last march, i think i might have been playing around with some annotation viewing feature... cannot recall. I just removed the dependency.

File: frontend/src/app/component/shared/time-element/time-element.component.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    } else {
+      this.ageTypes = ['Age', 'Age Range', 'Gestational Age'];
+    }
+    if (this.timeElement === undefined) {
+      this.timeElement = new TimeElement();
+    }
+    const element = this.timeElement.element;
+
+    if (element instanceof Age) {
+      this.selectedAgeType = this.ageTypes[0];
```
[the remark this answers is not in the document]
Comment (belkassaby) [reply frontend/src/app/component/shared/time-element/time-element.component.ts]: Fixed in [249c3](https://github.com/TheJacksonLaboratory/PhenopacketLab/pull/81/commits/249c3f0cf51879e936cb1fea022154693c9f46bf)

File: frontend/package.json [hunk 3]
```diff
@@ -32,16 +33,21 @@
...
     "bootstrap-material-design": "4.1.2",
     "chart.js": "^2.7.2",
     "core-js": "^3.20.0",
     "hammerjs": "~2.0.8",
     "perfect-scrollbar": "^1.4.0",
     "popper.js": "^1.16.1",
-    "ProtVista": "^2.0.13",
+    "primeflex": "^3.2.1",
+    "primeicons": "^6.0.1",
+    "primeng": "^14.1.2",
```
[the remark this answers is not in the document]
Comment (belkassaby) [reply frontend/package.json:49]: #94 will address the full material removal. For now its using both.

File: frontend/src/app/component/shared/time-element/time-element.component.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (belkassaby) [reply frontend/src/app/component/shared/time-element/time-element.component.ts]: Thx for the review btw Mike!

**Pull request merged**